cl/recap/tasks.py

```diff
@@ -1419,3 +1428,71 @@ def mark_fq_status(fq, msg, status):
     if status == PROCESSING_STATUS.SUCCESSFUL:
         fq.date_completed = now()
     fq.save()
+
+
+@app.task(bind=True, max_retries=5, interval_start=5 * 60)
+def process_recap_email(self: Task, pk: int) -> Optional[int]:
+    start_time = now()
+    epq = EmailProcessingQueue.objects.get(pk=pk)
+    mark_pq_status(epq, "", PROCESSING_STATUS.IN_PROGRESS, "status_message")
+
+    message_id = epq.message_id
+    bucket = RecapEmailSESStorage()
+    body = bucket.open(message_id, "r").read()
+    report = S3NotificationEmail(map_cl_to_pacer_id(epq.court_id))
+    report._parse_text(body)
+    data = report.data
+
+    if (
+        data == {}
+        or data.get("docket_entries", None) is None
+        or len(data["docket_entries"]) == 0
```

Comment:
I admit minor annoyance that this isn't tighter. If Juriscraper always sets the key to an empty list, we don't need `data.get("docket_entries", None) is None`. If OTOH, Juriscraper omits the key when there are no values, then we don't need the second line. We control Jurirscraper, right, so which is it? 

OTOH, I appreciate the defensive programming and if you ignore this comment completely, I'll live. :)

Reply:
That's a reasonable question @mlissner, and I'll admit to being overly defensive in my conditionals (my developers love to complain about this). Let me verify, and we will handle accordingly.